frontend/public/components/dashboard/dashboards-page/cluster-dashboard/details-card.tsx

```diff
@@ -205,6 +214,10 @@ export const DetailsCard = withDashboardResources(
                       {t('public~No (single master)')}
                     </DetailItem>
                   )}
+                  {detailItemsExtensions.map((e, index) => {
+                    const Component = e.properties.component;
+                    return <Component key={e.properties.id || index} />;
```

Comment:
why did you remove the `uid` usage ?
```suggestion
                    return <Component key={e.uid} />;
```

Reply:
done